rootfs/etc/confd/templates/builder

```diff
@@ -129,33 +129,35 @@ ACCESS_SECRET=`cat /var/run/secrets/object/store/access-secret-key`
 mkdir -p /var/minio-conf
 CONFIG_DIR=/var/minio-conf
 MC_PREFIX="mc -C $CONFIG_DIR --quiet"
-$MC_PREFIX config host add "$HTTP_PREFIX://$S3EP" $ACCESS_KEY $ACCESS_SECRET
-$MC_PREFIX mb "$HTTP_PREFIX://${S3EP}/git"
-$MC_PREFIX cp ${APP_NAME}.tar.gz $TAR_URL
-puts-step "stored tarfile in $TAR_URL"
+$MC_PREFIX config host add "$HTTP_PREFIX://$S3EP" $ACCESS_KEY $ACCESS_SECRET 2>&1
+$MC_PREFIX mb "$HTTP_PREFIX://${S3EP}/git" 2>&1
+$MC_PREFIX cp ${APP_NAME}.tar.gz $TAR_URL 2>&1
 
-puts-step "creating builder pod in namespace $POD_NAMESPACE"
-kubectl --namespace=$POD_NAMESPACE create -f /etc/${SLUG_NAME}.yaml
+kubectl --namespace=$POD_NAMESPACE create -f /etc/${SLUG_NAME}.yaml >/dev/null
+
+# wait for pod to be running and then pull its logs
+until [ "`kubectl --namespace=$POD_NAMESPACE get pods -o yaml ${META_NAME} | grep "phase: " | awk {'print $2'}`" == "Running" ]; do
```

Comment:
@gabrtv wants to see it written in !shell which I think is a good idea especially since we could integrate with kubernetes apis here for better? success?